packages/bus-query-operation/lib/ActorQueryOperation.ts

```diff
@@ -14,14 +13,6 @@ import type * as RDF from '@rdfjs/types';
 import type { Algebra } from 'sparqlalgebrajs';
 import { materializeOperation } from './Bindings';
 
-/**
- * A counter that keeps track blank node generated through BNODE() SPARQL
- * expressions.
- *
- * @type {number}
- */
-let bnodeCounter = 0;
```

Comment:
The fact that we can remove all of this stuff here is a sign we're on the right track :-)

Reply:
I just moved it somewhere else though :eyes:

Reply:
The logic is fine, but the location was not :-)